lib/src/toggle-group/types.ts

```diff
@@ -0,0 +1,73 @@
+type Space = "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | "xxlarge";
+type Margin = {
+  top?: Space;
+  bottom?: Space;
+  left?: Space;
+  right?: Space;
+};
+type SVG = React.SVGProps<SVGSVGElement> | React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
+
+type Option={
+    /**
+    * Number with the option inner value.
+    */
+    value?: string;
+    /**
+    * String with the option display value.
+    */
+    label?: string;
+    /**
+    * Element used as the icon. Icon and label can't be used at same time.
+    */
+    icon?: SVG;
+    /**
+    * @deprecated URL of the icon that will be placed. IconSrc and label can't be used at same time.
+    */
+    iconSrc?: string;
+};
+
+type Props = {
+  /**
+   * Text to be placed above the component.
+   */
+  label?: string;
+  /**
+   * Helper text to be placed above the component.
+   */
+  helperText?: string;
+  /**
+   * The key(s) of the selected value(s). If the toggle group component doesn't allow multiple selection, 
+   * it must be one unique value. If the component allows multiple selection, value must be an array. 
+   * If undefined, the component will be uncontrolled and the value will be managed internally by the component.
+   */
+  value?: string | string[];
+  /**
+   * This function will be called every time the selection changes. The number with the key of the selected 
+   * value will be passed as a parameter to this function. 
+   * If multiple selection is allowed, an array of keys will be passed.
+   */
+  onChange?:(optionIndex: number | number[]) => void;
```

Comment:
I think we should talk with @jsardid and @aweell about the `onChange`. Perhaps it makes sense to follow the same approach as Select and use the `value: string | string[]` as the parameter of this event.